swagger.yml

```diff
@@ -125,6 +234,29 @@ definitions:
             type: string
           reviews:
             type: string
+  Review:
+    type: object
+    required:
+      - message
+      - user
+    properties:
+      id:
+        description: "Unique book id"
+        type: string
+      last_updated:
+        description: "UTC timestamp of when the review was last updated"
+        type: string
+      message:
+        description: "Review message from user"
+        type: string
+      user:
+        description: "Reviewer details"
+        type: object
```

Comment:
We can add the following keyword for a user, `required: [forenames, surname]` to identify these fields as being mandatory in the `user` object.

Reply:
This can be structured like you have previously done for `Review` object:

```
required:
  - forenames
  - surname
```